test/unit/commands/app/create.test.ts

```diff
@@ -17,9 +17,7 @@ const { origin, pathname } = new URL(config.appBoilerplateGithubUrl);
 const zipPath = join(process.cwd(), "test", "unit", "mock", "boilerplate.zip");
 const region: { cma: string; name: string; cda: string } =
   configHandler.get("region");
-const developerHubBaseUrl = (config.developerHubUrls as Record<string, any>)[
-  region.cma
-];
+const developerHubBaseUrl = configHandler.get("developerHubBaseUrl");
```

Comment:
@harshithad0703 , Could you please verify that you are receiving the developerHubBaseUrl value by using configHandler.get("developerHubBaseUrl")? As far as I'm aware, we are not configuring it in the config file.

Pls check base-command code how are we storing developerHubBaseUrl.